### PR TITLE
Write updated modal settings to url

### LINKED ISSUE #3338: [low prio] incorrectly rendered rounding when setting the size via updateLuigiModalSettings
When setting the size via updateLuigiModalSettings to 'l', we receive an incorrectly rendered rounding in the modal

image

If I set the size to 'l' in the URI parameter, the rounding is rendered correctly.

Before:
image

After setting size to 'l' in URI parameter:
 
image

_To reproduce, open an arbitrary template in the Hyperspace Portal(1. Create a project or open one where you are a member 2. Click Components 3. Click Create 4. Select an arbitrary template)_

### PR DESCRIPTION
**Pull request by JohannesDoberer**
Fixes #3338

### CODE REVIEW

File: client/src/linkManager.js [hunk 1]
```diff
@@ -170,18 +170,21 @@ export class linkManager extends LuigiClientBase {
   }
 
   /**
-   * Update current title and size of a modal.
+   * Update current title and size of a modal. If `routing.showModalPathInUrl` is set to true, the URL will be updated with the modal settingds data.
```
Comment (alexandra-simeonova) [inline client/src/linkManager.js:173]:
```suggestion
   * Updates the current title and size of a modal. If `routing.showModalPathInUrl` is set to `true`, the URL will be updated with the modal settings data.
```

File: client/src/linkManager.js [hunk 2]
```diff
@@ -170,18 +170,21 @@ export class linkManager extends LuigiClientBase {
...
+   * In addition you can specify if a new history entry wil lbe created with the updated URL.
```
Comment (alexandra-simeonova) [inline client/src/linkManager.js:174]:
```suggestion
   * In addition, you can specify if a new history entry will be created with the updated URL.
```

File: core/src/Modal.svelte [hunk 3]
```diff
@@ -147,6 +149,7 @@
       height = settingsHeight;
       width = settingsWidth;
     } else {
+      elem.classList.remove('lui-modal-fullscreen');
```
Comment (hardl) [inline core/src/Modal.svelte:152]: why only remove in the else block?

**Review by hardl — CHANGES_REQUESTED** [review]

**Review by hardl — APPROVED** [review]
LGTM

**Pull request merged**